### PR TITLE
Increase the size of the console output.

### PR DESCRIPTION
**Pull request by nicolasnoble**

### CODE REVIEW

File: tools/run_tests/jobset.py [hunk 1]
```diff
@@ -210,7 +210,7 @@ def state(self, update_cache):
       # TODO: looks like jenkins master is slow because parsing the junit results XMLs is not
```
Comment (jtattermusch) [inline tools/run_tests/jobset.py]: how about we update the comment to say that it's not the main cause of slowness but we still want to make sure we don't store overly huge logs.

Btw, do you think that an average stacktrace would be about 2KB in size? My very rough guess would be that it's a bit bigger. Let's try 4-10KB?

**Comment by jtattermusch** [comment]
is this PR still alive?

**Comment by nicolasnoble** [comment]
Ah, sorry, didn't notice the comment.

**Comment by ctiller** [comment]
Not mergable

**Comment by jtattermusch** [comment]
Superseded by #4008

**Pull request closed without merging**